src/views/RecIM/components/Header/index.js

```diff
@@ -24,22 +21,27 @@ const RecIMBreadcrumb = ({ link, children }) => {
   return <Typography color="text.primary">{children}</Typography>;
 };
 
-const Header = ({ page, expandable = false, match, team, activity, home, admin }) => {
+const Header = ({
+  page,
+  expandable = false,
+  match,
+  team,
+  activity,
+  home,
+  admin,
+  setOpenHeaderForm,
```

Comment:
All forms simply are just setOpen____Form so this would just make it more simple. As opposed to a bunch of states.